requirements.txt

```diff
@@ -1,2 +1,2 @@
 numpy >= 1.6
-GDAL >= 1.11
\ No newline at end of file
+GDAL < 2.0.0
```

Comment:
huh?  why is this changing?  And why wouldn't GDAL 2.0.0 work?  (we are using 2.1dev in planet_common)